tracing-appender/src/rolling.rs

```diff
@@ -34,7 +34,7 @@ use std::{
     path::{Path, PathBuf},
     sync::atomic::{AtomicUsize, Ordering},
 };
-use time::{format_description, Duration, OffsetDateTime, Time};
+use time::{format_description, Date, Duration, OffsetDateTime, Time};
```

Comment:
if you take my suggestion about using the file's creation timestamp rather than parsing the date from the filename, we can remove this:
```suggestion
use time::{format_description, Duration, OffsetDateTime, Time};
```